src/components/Card.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { ICardProps } from '../models/propsModels'
+
+export const Card = ({ promo }: ICardProps) => {
```

Comment:
This component use properties only from promo object. For rendering purposes (when react decided should it update DOM tree or not with new props) React uses shallow comparison that leads to unnecessary renders with object type of props.

To avoid this and get maximum of performance we should accept only plain types for properties.

Instead of
```tsx
const  Card = ({ promo }: ICardProps) => (
  <p>{promo. description}</p>
);

const Container = () => (
  <Card promo={{ description: 'One more thing about React' }} />
);
```

We can refactor it
```tsx
interface ICardProps {
  description: string;
}

const Card = ({ description }: ICardProps) => (
  <p>{description}</p>
);

const Container = () => (
  <Card description='One more thing about React' />
);
```

And to simplify passing multiple props we can use syntax sugar.
```tsx
const Container = () => (
  <Card {...promo} />
);
```